TheFourFifths/consus
# [TFF-170] Fix `consusd` and allow port selection for integration tests

### Summary

This pull request enables the integration test suite to start the server as a background process on a specific port.

Sister PRs (merge **this** PR first!):
* TheFourFifths/consus-client/pull/90 (merge second!)
* TheFourFifths/consus-integration/pull/2 (merge last!)

### Checklist


### How to Review

1. Check out the `functional-integration-TFF-170` branch on `consus`
2. `npm install -g`
3. `consusd start --port=8081`
4. Verify that the server is listening by opening http://localhost:8081/api/item/all
5. `consusd stop`
6. Verify that the server is no longer listening

### Links

- [TFF-170](https://msoese.atlassian.net/browse/TFF-170)
- TheFourFifths/consus-integration/pull/2
- TheFourFifths/consus-client/pull/90
